explanation/explanation.py

```diff
@@ -1,3 +1,41 @@
-class Explanation():
-    def __init__(self, meta_paths, domain_value):
+from util.datastructures import MetaPath, MetaPathRating
+
+import numpy as np
+
+# mock data
+mock_paths = [MetaPathRating(MetaPath([1, 2], [3]), 3, 1), MetaPathRating(MetaPath([1, 2, 1], [3, 4]), 1, 3),
```

Comment:
Is it necessary to have the mock-data here and in the tests?

Reply:
nope, deleted it now